eng/common/scripts/SemVer.ps1

```diff
@@ -81,6 +90,15 @@ class AzureEngSemanticVersion {
         $this.PrereleaseNumberSeparator = $matches["prenumsep"]
         $this.IsPrerelease = $true
       }
+      if ($null -eq $matches["buildnumber"]) {
+        $this.HasBuildNumber = $false
+      } else {
+        $this.BuildNumberSeparator = $matches["buildnumsep"]
+        if ($matches.ContainsKey("buildnumber")) {
```

Comment:
Is this contains necessary based on the $null check above?

Reply:
Ah, holdover from before I added the conditional variable.